### PR TITLE
Awakened cat/wolf-person profession

### PR DESCRIPTION
**Pull request by onura46**
#### Summary
Content "Adds 'Awakened Cat/Wolf Boy/Girl' scenario and mission to actually become a cat/wolf-person"

#### Purpose of change
Adds the "Awakened Cat/Wolf Boy/Girl" professions for some fun roleplaying. 

#### Describe the solution
The Awakened Cat/Wolf-Person starts with a fursuit of the appropriate type, a burning desire to mutate into a cat/wolf, and little else. Consuming mutagen of the appropriate type will fulfill the mission, but random mutations _specifically_ of the feline/lupine type will also fulfill it. So, if you just sit around a bunch of toxic waste dreaming of becoming a cat-person, it might happen! +1 for self-actualization!

#### Describe alternatives you've considered
None

#### Testing
Tested on my compiled version with the newest updates.

### CODE REVIEW

**Comment by Maleclypse** [comment]
I'm gonna be honest I thought this going to go in a totally different direction and I am pleasantly surprised.

File: data/json/professions.json [hunk 1]
```diff
@@ -5060,5 +5060,33 @@
       ]
     },
     "missions": [ "MISSION_ASSASSINATION" ]
+  },
+  {
+    "type": "profession",
+    "id": "awakened_cat",
+    "name": { "male": "Awakened Cat Boy", "female": "Awakened Cat Girl" },
+    "description": "Just yesterday you were at a convention with your friends.  Now, the world is falling apart.  Though you long to see them again, something about the collapse of society unlocks a deep, feline desire to explore!  And maybe even become what you've secretly wanted all your life...",
```
Comment (Maleclypse) [inline data/json/professions.json:5068]:
```suggestion
    "description": "Just yesterday you were at a convention with your friends.  Now, the world is falling apart.  Though you long to see them again, something about the collapse of society unlocks a deep, feline desire to explore!  And maybe even become what you've secretly wanted all your life…",
```

File: data/json/professions.json [hunk 2]
```diff
@@ -5060,5 +5060,33 @@
...
+      "male": [ "boxer_shorts", "tank_top" ],
+      "female": [ "bra", "panties" ]
+    },
+    "missions": [ "MISSION_BECOME_CAT" ]
+  },
+  {
+    "type": "profession",
+    "id": "awakened_wolf",
+    "name": { "male": "Awakened Wolf Boy", "female": "Awakened Wolf Girl" },
+    "description": "Just yesterday you were at a convention with your friends.  Now, the world is falling apart.  Though you long to see them again, something about the collapse of society unlocks a deep, canine desire to run free and find your pack!  And maybe even become what you've secretly wanted all your life...",
```
Comment (Maleclypse) [inline data/json/professions.json:5082]:
```suggestion
    "description": "Just yesterday you were at a convention with your friends.  Now, the world is falling apart.  Though you long to see them again, something about the collapse of society unlocks a deep, canine desire to run free and find your pack!  And maybe even become what you've secretly wanted all your life…",
```

**Comment by TheShadowFerret** [comment]
Dino Dave wouldn't be mad if there was an Awakened Dino Guy/Girl in Dinomod, right?

**Comment by NetSysFire** [comment]
You need to ask @LyleSY for that, but its a fun idea for sure

**Comment by KittyTac** [comment]
lmao. I always wanted a "Furry" profession but this is even better.

**Comment by faefux** [comment]
This should be Cat /Wolf Man/Woman. r even better Cat/Wolf Person. Just to be correct!

**Comment by sonphantrung** [comment]
No shark boys/girls D8

**Comment by onura46** [comment]
> This should be Cat /Wolf Man/Woman. r even better Cat/Wolf Person. Just to be correct!

Not trying to defend anything here really, but I chose the wording I did because it shows up on your character sheet like that: "Lyle Greenspan - Awakened Cat-Boy". And I thought that was kinda cute and fitting to the 'uWu rawr' vibe I was trying to inspire with the descriptions. I'm not tied to it, but I would definitely prefer to use true non-binary options once they exist like the gendered options exist now. The _very moment_ someone switches the Cata gender value from a boolean to an int, I'll be back to clean this up, I promise <3

> No shark boys/girls D8

I _can't_ really justify leaving out anyone's particular anthropomorphic wishes here, other than to say I didn't want to overwhelm the professions list off the back of one PR or add any new items to the game 😅 I just went with the most "popular" options, so to speak. Should this PR get merged, I'll see about expanding it based on feedback a bit later.

**Comment by descan** [comment]
If only it were possible to hide certain sub-professions until after you selected the main category, then you could have a plethora of furry options under "Awakened Anthro" (Or something) with every specific possible mutation category as a choice, without having it clog the professions list. (At least the ones that make sense. I can see a plant-furry, but a "medical" furry? not so much).

Of course, there are furries that go after species that aren't mutatable at present, and while there might be some funny existential drama behind wanting to fur-up into a species you physically can't in the game, it wouldn't be very satisfying to play as that.

**Comment by Maleclypse** [comment]
> 

I think the proficiency system does that where you can set some background and a mission.

**Comment by LyleSY** [comment]
I would be open to this in DinoMod if you want to contribute it there. The current dino mutation trees are stegosaurus, raptor, and tyrannosaurus. I'd also be open to more mutation trees if you're interested in that. Also there is a basic DinoLab that has these mutagens available but it needs more work to make it feel custom and interesting. There is also an existing dino mutant NPC with a quest chain around mutation.

**Comment by descan** [comment]
> > 
> 
> I think the proficiency system does that where you can set some background and a mission.

Aye, that could work. "Awakened Anthro/Furry" as a profession, and a background setting of which species you emulate. 

Whether the backgrounds should be 0 point or not, and if not 0 point if they should be differently pointed, is another question... On the one hand, some mutation categories are more powerful than others - some categories are harder to get into safely than others. On the other hand, which mutation category is "the best" or "the worst" or "mediocre" can also be extremely subjective, which ranking based on point cost would assign objective values to. "These mutation categories are The Best so they are the most expensive." 

And if they aren't differently pointed, they should cost 0 because the cost would be baked into the profession choice cost, if they is one.

(Admittedly, maybe this shouldn't so much be a profession in the first place as it should be a background? Furries aren't exactly a Job you do, they're something anyone can do - firefighters, bionic soldiers, tweakers, hackers, they can all be furries. SOMEONE has to be extremely rich and successful in their job of Not Just Being A Furry to pay all those furry artists, after all!)

**Comment by onura46** [comment]
> Admittedly, maybe this shouldn't so much be a profession in the first place as it should be a background? 

The only problem with classifying it as a background is that I can't give the player any items to start, like a particular fursuit. Or maybe you can and I'm mistaken? I agree that it would fit backgrounds a bit better, yeah.

**Comment by descan** [comment]
> > Admittedly, maybe this shouldn't so much be a profession in the first place as it should be a background?
> 
> The only problem with classifying it as a background is that I can't give the player any items to start, like a particular fursuit. Or maybe you can and I'm mistaken? I agree that it would fit backgrounds a bit better, yeah.

Mmm, that would be an issue. Based purely on the player-facing UI, it looks like they're restricted to Addictions, Traits, Proficiencies, and Skill Experience. Unless a) they give them to you but you can only see that by going back to professions and checking, like with certain traits (far/near sighted giving you the correct glasses, for example - the trait tells you, but you can only see the item listed in the Professions tab) or b) the background traits are coded to allow giving items, but none of them as-yet actually does so (which seems... unlikely.) 

Checking through a number of candidates (the dependencies, skating, cooking, backpacking, knife throwing) that *could* easily have given you the correct item to start with indicates that a) is not the case. So unless b) is the case, backgrounds would need to be changed to allow giving of items.

**Comment by kevingranade** [comment]
I have several problems with this, tl;dr is "no".
From a project tone point of view, we certainly have content that can be viewed as being furry friendly, but that's more not going out of our way to exclude things because they're furry-adjacent rather than active promotion.  This PR takes that into actively endorsing it, it's not a game about being a furry, it's a game about survival.
From an internal consistency point of view, this jumps straight from someone having furry tendencies to going all-in on trying to literally change into their fursona, which as far as that character knows is impossible, so it's a nonsensical goal to have.
Finally, it is very much cannonical that mutated humans don't remotely resemble most fursonas, DDA mutants are far more in the realm of Dr. Moreau mutants than cartoon anthro characters.

**Pull request closed without merging**